src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -62,7 +62,9 @@ export default function appointmentsReducer(state = initialState, action) {
     case FETCH_FUTURE_APPOINTMENTS:
       return {
         ...state,
-        pendingStatus: FETCH_STATUS.loading,
+        pendingStatus: action.includeRequests
```

Comment:
Don't want to set the pending status to loading if we're not fetching requests, if we do then requests aren't fetched on the requested page